nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseRecord.java

```diff
@@ -127,6 +127,18 @@
             .defaultValue("1000")
             .build();
 
+    protected static final AllowableValue NULL_FIELD_EMPTY = new AllowableValue("empty-string", "Empty String", "Use an empty string");
+    protected static final AllowableValue NULL_FIELD_SKIP  = new AllowableValue("skip-field", "Skip Field", "Skip the field (don't process it at all).");
+
+    protected static final PropertyDescriptor NULL_FIELD_STRATEGY = new PropertyDescriptor.Builder()
+            .name("hbase-record-null-field-strategy")
+            .displayName("Null Field Strategy")
+            .required(true)
+            .defaultValue("empty-string")
```

Comment:
IMHO, I personally prefer having 'Skip Field' as the default value, because I think this processor is used more for ingesting stream of data, than updating existing data. In that sense, I prefer not to store empty bytes to minimize resource utilization.